client.go

```diff
@@ -11,14 +11,17 @@ import (
 )
 
 var (
-	version = "v1.0.0"
-	AuthURL = url.URL{Scheme: "https", Host: "auth.hashicorp.com", Path: "/oauth/token"}
-	BaseURL = url.URL{Scheme: "https", Host: "api.cloud.hashicorp.com", Path: "/secrets/2023-06-13"}
+	version       = "v1.1.0"
+	AuthURL       = url.URL{Scheme: "https", Host: "auth.hashicorp.com", Path: "/oauth/token"}
+	BaseURL       = url.URL{Scheme: "https", Host: "api.cloud.hashicorp.com", Path: "/secrets/2023-06-13"}
+	HCPOrgURL     = url.URL{Scheme: "https", Host: "api.hashicorp.cloud", Path: "/resource-manager/2019-12-10/organizations"}
+	HCPProjectURL = url.URL{Scheme: "https", Host: "api.hashicorp.cloud", Path: "/resource-manager/2019-12-10/projects"}
 )
 
 type Client struct {
-	OrganizationID  string
-	ProjectID       string
+	OrganizationID  string `json:"organization_id,omitempty"`
```

Comment:
The client isn't something that should be processing JSON as a response itself, so I wouldn't expect to see `json` tags in this struct.